tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -187,8 +187,32 @@ def _run(self):
                                         hostname not in network_ip):
                                         b_index += 1
                                         continue
+                                    sub_vol_size = (int(volumes['volume%s.brickcount' % index])) / int(
```

Comment:
Gluster vol should have at least 1 brick to exist. Hence sub-vol count will be at least 1.